src/main.js

```diff
@@ -0,0 +1,11 @@
+import Vue from 'vue'
+import App from './App.vue'
+import dotenv from 'dotenv'
```

Comment:
Absolute imports should come before relative imports  import/imports-first<br>Missing semicolon                                     semi